frontend/src/lib/components/LemonSelect.tsx

```diff
@@ -30,6 +30,9 @@ export interface LemonSelectProps<O extends LemonSelectOptions>
     dropdownMaxContentWidth?: boolean
     dropdownPlacement?: PopupProps['placement']
     allowClear?: boolean
+    className?: string
+    popupClassName?: string
+    popupRef?: React.MutableRefObject<HTMLDivElement | null>
```

Comment:
I think, according to what's described in [this long article](https://www.gabe.pizza/notes-on-component-libraries/) under "A component interface concisely representing the variants of the design system is easier to use, it's best to pass in `popup = {ref,className,...}` and send that on directly, instad of `popupClassName`...

(happy to leave it for another day)